cooperative_lanechange/src/cooperative_lanechange.cpp

```diff
@@ -820,8 +820,6 @@ namespace cooperative_lanechange
 
         int total_points = std::min(centerline_start_lane.size(), centerline_end_lane.size());
         double delta_step = 1.0/(total_points - 1);
-        
-        ROS_DEBUG_STREAM(">>>>>> Creating lanechange geom");
 
         for(int i=0; i<total_points;i++){ // start from 1 as star_lane_pt is included
```

Comment:
Looks like this comment needs to be updated because index starts from 0 and comment is not correct.  Remove it.